crt0/switch_crt0.s

```diff
@@ -3,8 +3,10 @@
 
 _start:
     bl startup
+    .word 0
     .ascii "HOMEBREW"
 
+.org 0x80
```

Comment:
Strictly speaking this should either be `.org _start+0x80`, or using a `.space` directive.